examples/mistral/mistral_optimize.json

```diff
@@ -0,0 +1,128 @@
+{
+    "input_model": {
+        "type": "PyTorchModel",
+        "config": {
+            "hf_config": {
+                "model_name": "mistralai/Mistral-7B-v0.1",
+                "model_class": "MistralForCausalLM"
+            }
+        }
+    },
+    "systems": {
```

Comment:
Is this required? I think it's cleaner if we just specify ep as `CPUExecutionProvider` under engine so that it automatically infers the system as cpu?

Reply:
I put it here for demo purpose in case of user want to run on GPU by simply change the accelerators to gpu.